### DISCUSSION TITLE
Is it necessary to use migration stubs?

### DISCUSSION THREAD

**Discussion by varemel** [def-studio/telegraph#414] [General]
Hello,

Kindly appreciate your work. 

Can you please explain your thoughts behind choosing migration stub `create_telegraph_bots_table.php.stub` rather than a regular migration file `create_telegraph_bots_table.php`?
As I can understand stubs needed to create template. But I can't see any template args in your stub files.

![005640](https://github.com/defstudio/telegraph/assets/84812777/73a1771a-f9c6-4f7b-83f5-cc2b82b062db)

**Comment by fabio-ivona** [comment]
Hi! thanks for your appreciation, it means a lot for us

Stubs are a practice that allows to have php files without being parsed by your IDE, as they are basically excluded from project until you publish migrations
